libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -165,6 +166,9 @@ export class DtRadialChart implements AfterContentInit, OnDestroy {
   }
 
   constructor(
+    // TODO: remove this sanitizer when angular@10
+    // and use the following syntax on template [style.--var]="value"
+    public sanitizer: DomSanitizer,
```

Comment:
@dynatrace-oss/barista I guess this is considered a breaking change. Correct me if i am wrong here. 
Could you also move it to the end of the constructor please and make it private :)